testing/cloud.py

```diff
@@ -47,6 +47,20 @@ def __getattr__(self, name):
         return os.environ[name]
 
 
+def get_resource_path(resource, local_path):
```

Comment:
Oh I see what the difference is. I didn't realize that you weren't *necessarily* getting the resource from the local path. I see where the awkwardness lies now.

Okay - it's fine the way it is. Suggestion if you like it:

    def get_resource_path(*resource, local_path=None):

and add a check for `local_path` before using it. That feels marginally more semantic for me, but whatever.